app/Controller/PagesController.php

```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
+public function display1() {
+}
+public function display2() {
```

Comment:
__[staging][code_sniffer]__
* Line indented incorrectly; expected 4 spaces, found 0
* Missing function doc comment
* Opening brace should be on a new line